mdx/api/index.mdx

```diff
@@ -415,6 +415,62 @@ Use "ETH" as the `sellToken` to receive a [Forwarder](/docs/guides/v3-forwarder-
 ```
 </CodeTabs>
 
+#### Wrapping and unwrapping between ETH and WETH
+
+Easily wrap and unwrap between ETH and WETH by requesting a swap quote by setting `sellToken` and 'buyToken' as either 'WETH' or 'ETH'. The swap quote returned will provide the calldata to directly interact with the `WETH9` contract and not with 0x exchange contracts.
+
+##### Wrap ETH
+
+<HttpCall fontSize="16px" path="/swap/v0/quote?buyToken=ETH&sellToken=WETH&buyAmount=10000000" />
+
+<CodeTabs tabs={['200 - Wrapping Transaction']}>
+
+```json
+{
+    "price": "1",
+    "guaranteedPrice": "1",
+    "to": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+    "data": "0x2e1a7d4d0000000000000000000000000000000000000000000000000000000000989680869584cd0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000005e7cf06f",
+    "value": "0",
+    "gas": "0",
```

Comment:
This should either be non-zero or not present in the response, it should never be 0. 

Should be ~ 60k for a withdraw, ~ 40k for a deposit